src/variable/VRLadle.sol

```diff
@@ -338,20 +318,19 @@ contract VRLadle is UUPSUpgradeable, AccessControl() {
         if (ink != 0) {
             IJoin ilkJoin = getJoin(vault.ilkId);
             if (ink > 0) ilkJoin.join(vault.owner, uint128(ink));
-            if (ink < 0) ilkJoin.exit(to, uint128(-ink));
+            else ilkJoin.exit(to, uint128(-ink));
         }
 
         // Manage base
         if (base != 0) {
             IJoin baseJoin = getJoin(vault.baseId);
             if (base < 0) baseJoin.join(vault.owner, uint128(-base));
-            if (base > 0) baseJoin.exit(to, uint128(base));
+            else baseJoin.exit(to, uint128(base));
```

Comment:
What if `base` is 0? That wasn't being checked before but is now?

Reply:
On 325 we are checking that `base != 0`